.github/workflows/_build-wheel-release-upload.yml

```diff
@@ -123,6 +123,14 @@ jobs:
           body_path: CHANGELOG.txt
           token: ${{ secrets.GITHUB_TOKEN }}
 
+  docs:
```

Comment:
Problem 1. 

As we've encountered with `diffpy.utils`, when we do a full pre/full release, the reusable doc CI isn't triggered no matter what (I tried a bunch including `created`, `published`, `prelreased`, etc)

A solution is to ensure we build it during the release proceed, while allowing the docs to be deployed by workflow dispatch as usual. 

(I've tested this using my dummy org/repo)